contracts/MarketCollateralised.sol

```diff
@@ -68,19 +69,34 @@ abstract contract MarketCollateralised is Market {
 		bytes16 propositionId
 	) internal virtual returns (uint256) {
 		uint256 result;
+
 		uint256 internalCollateralToUse;
```

Comment:
Can we move this var creation to LN: 79?

Reply:
I don't follow

Reply:
The var internalCollateralToUse isn't used to after https://github.com/horse-link/contracts.horse.link/pull/99/files#diff-f5900bd5f7e2d9283f2b98c019a41f01b7792490e99c877ac08b20c7106e4acfR78

Reply:
> I don't follow

Lets move it to inside the "if" condition on LN78